configuration.json

```diff
@@ -1,94 +1 @@
816433a9b6e6690b0b8e919ffa874982706e812314aae09d85fc62fc4fa3c","entity3Pubk":"0xf6aa52924a7280241bd84d098d7c03b4e3a7e08f206b68868f439b65a6c3b26b1bd30b960325be8670339a66258b851ae170691429248149b601da5798f42d28","subject1Pubk":"0x2e507af01167c98a6accc0cd46ab2a256dd6b6c69ec1c0c28f80fb62e1f7d70233768b0c58dbbdac1fc358b8141c075a520483cf9779e4ea98d13df2833f3767","subject2Pubk":"0xaca7aa42e831f41b6f246d089fbcabe73e1c6ce01a1bb5d944fda4e148e1d6930f6ebb5138b1d103e3e384c893962a238e4088a549bfe84867da96942931f69a","rawPrivateKey":"278a5de700e29faae8e40e366ec5012b5ec63d36ec77e8a2417154cc1d25383f","context":"https://w3id.org/did/v1","userPublicKey":"AE2309349218937HASKHIUE9287432","providerURL":"https://regular.telsius.blockchainbyeveris.io:2000","callbackURL":"https://serviceprovider.alastria.blockchainbyeveris.io/api/login/","alastriaNetId":"Alastria network","tokenExpTime":1563783392,"tokenActivationDate":1563782792,"jsonTokenId":"ze298y42sba","network":"quor","networkId":"redT","tokenPayload":{"iss":"joe","exp":1300819380,"http://example.com/is_root":true},"jti":"https://www.empresa.com/alastria/credentials/3734","kidCredential":"did:ala:quor:redt:QmeeasCZ9jLbXueBJ7d7csxhb#keys-1","subjectAlastriaID":"did:alastria:quorum:redt:QmeeasCZ9jLbXueBJ7d7csxhb","credentialKey":"StudentID","credentialValue":"11235813","uri":"www.google.com","procUrl":"https://www.empresa.com/alastria/businessprocess/4583","procHash":"H398sjHd...kldjUYn475n","data":[{"@context":"JWT","levelOfAssurance":"High","required":true,"field_name":"name"},{"@context":"JWT","levelOfAssurance":"High","required":true,"field_name":"email"}],"entity1":"0xb238ce01fa350387cd33d7007709422c521a0126","didEntity1":"did:ala:quor:redT:b238ce01fa350387cd33d7007709422c521a0126","entity2":"0x589a3be55729d60a39fdba8b591c1ab24d727c11","didEntity2":"did:ala:quor:redT:589a3be55729d60a39fdba8b591c1ab24d727c11","entity3":"0x9cd913ea94a812f445ec9be8452915d54f05b366","didEntity3":"did:ala:quor:redT:9cd913ea94a812f445ec9be8452915d54f05b366","subject1":"0x87ff106536c5c129e8a5e1434888eb1c5fb5a927","didSubject1":"did:ala:quor:redT:87ff106536c5c129e8a5e1434888eb1c5fb5a927","subject2":"0x5a6d79a6240ce5724f551922a2d1addee9fcfa4c","didSubject2":"did:ala:quor:redT:5a6d79a6240ce5724f551922a2d1addee9fcfa4c","issuerLevel":2,"updateSubject1PresentationTo":2,"updateEntity2PresentationTo":1,"updateIssuerCredentialTo":2,"subject1PresentationStatus":{"exist":true,"status":"2"},"entity2PresentationStatus":{"exist":true,"status":"1"},"entityData1":{"name":"Entidad1","cif":"A-28964782","urlLogo":"www.entidad1.com/logo","urlCreateAID":"www.entidad1.com/AID","urlAOA":"www.entidad1.com/AOA","status":true},"entityData2":{"name":"Entidad2","cif":"A-95974786","urlLogo":"www.entidad2.com/logo","urlCreateAID":"www.entidad2.com/AID","urlAOA":"www.entidad2.com/AOA","status":true},"entityData3":{"name":"Entidad3","cif":"A-49934581","urlLogo":"www.entidad3.com/logo","urlCreateAID":"www.entidad3.com/AID","urlAOA":"www.entidad3.com/AOA","status":true}}
```

Comment:
Esto, si no es mucha molestia, dar formato para que se pueda leer mejor por los desarrolladores, por ejemplo como está ahora en develop. No sé si me explico ...

Reply:
totalmente de acuerdo, pero el tema es que cuando ejecuto los examples, al final de ellos se ejecuta un JSON.stringify sobre el mismo fichero y lo reformatea siempre, cada vez que ejecuto. ¿Cómo lo resolvéis? ¿Manualmente lo reformateáis antes de volver a subirlo?

Reply:
Como se comentó en una reunión, nosotros lo reformateamos a mano, usando Visual Studio Code o similar.